docs/integrations/destinations/bigquery.md

```diff
@@ -111,15 +111,13 @@ This is the recommended configuration for uploading data to BigQuery. It works b
 * **GCS Bucket Path**
 * **Block Size (MB) for GCS multipart upload**
 * **GCS Bucket Keep files after migration**
-  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket.
+  * See [this](https://cloud.google.com/storage/docs/creating-buckets) for instructions on how to create a GCS bucket. The bucket cannot have a retention policy. Set Protection Tools to none or Object versioning.
 * **HMAC Key Access ID**
-  * See [this](https://cloud.google.com/storage/docs/authentication/hmackeys) on how to generate an access key.
-  * We recommend creating an Airbyte-specific user or service account. This user or account will require read and write permissions to objects in the bucket.
+  * See [this](https://cloud.google.com/storage/docs/authentication/managing-hmackeys) on how to generate an access key. For more information on hmac keys please reference the [GCP docs](https://cloud.google.com/storage/docs/authentication/hmackeys)
+  * We recommend creating an Airbyte-specific user or service account. This user or account will require the following permissions for the bucket: `Storage Object Admin`. You can set those by going to the permissions tab in the GCS bucket and adding the appropriate the email address of the service account or user and adding the aforementioned permissions.
```

Comment:
is there a reason we didn't originally specify the the exact permissions needed? the GCP permission model has so many permissions here that seem vaguely related. it took me a few tries to get the right one. maybe there's another combination of `Legacy Storage Reader`, etc, that works as well? If so maybe we should document it as well.